rpcs3/Emu/Cell/Modules/cellSysutil.cpp

```diff
@@ -413,6 +413,90 @@ s32 cellSysutilGameReboot_I()
 	fmt::throw_exception("Unimplemented" HERE);
 }
 
+s32 _ZN4cxml7Element11AppendChildERS0_()
```

Comment:
These mangled names are horrid. Can't they be hidden behind a macro so the emu code can call and use the unmangled name?